index.html

```diff
@@ -26,13 +26,14 @@
     <section class="home" id="home">
 
       <!-- home -->
-      <div class="home">
+      <div class="grediant">
+        
+        <!-- <div style="background-image: url('images/back.jpg'); width="100%" ,height="900px"> -->
```

Comment:
Please remove unwanted comments in html file

Reply:
done